backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/crowd/service/CrowdAuthenticationService.kt

```diff
@@ -7,8 +7,7 @@ import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
 import java.util.UUID
 
-@Service
-class CrowdAuthenticationService @Autowired constructor(private val crowdClient: CrowdClient) : ExternalAuthenticationService {
+class CrowdAuthenticationService constructor(private val crowdClient: CrowdClient) : ExternalAuthenticationService {
```

Comment:
I think you still need the `@Autowired`. Or not?

Reply:
Not; if the Configuration class contructor where the CrowdAuthenticationService is created is Autowired. (But that may change.)

Reply:
Oh, yes, of course. You're completely right :)